starter-code/src/assessment1.js

```diff
@@ -1,36 +1,69 @@
 // Write a function that returns the product of 2 numbers
 function product(x, y) {
+  return x * y;
   //
 }
 
 // Write a function that returns whether a given number is even
 function isEven(num) {
+  if (num%2 == 0) {
+    return true;
+  } else {
+    return false;
+  }
   //
 }
 
 // Return the largest of 2 numbers
 function maxOfTwoNumbers(a, b) {
+  if (a > b)
+    return a;
+  else
+    return b;
   //
 }
 
 // Return the largest of 3 numbers
 function maxOfThreeNumbers(a, b, c) {
-  //
-}
+			if(a > b && a > c)
+				{
+					return a;
+				}
+			else if(b > a && b > c)
+				{
+					return b;
+				}
+			else if(c > a && c > a)
+				{
+					return c;
+        }
+
+      }  //
 
 // Calculate the sum of an array of numbers
 function sumArray(numbers) {
-  //
+  if(!Array.isArray(numbers)) return;
+  let sumArray = 0, i=-1;
+  while (++i < numbers.length) {
+     sumArray+=numbers[i];
+  }
+  return sumArray;
 }
 
 // Return the largest number of a non-empty array
 function maxOfArray(numbers) {
-  //
+ //
```

Comment:
you can set a var to be the first number in array (if there is something in array) and run a loop to check which number is greater (num in the array or the var), if its num in array then the var now becomes the num in array and return that var at the end